pkg/ext-proc/backend/suite_test.go

```diff
@@ -0,0 +1,20 @@
+package backend
```

Comment:
do we need this file?

Reply:
I would recommend keeping the klog in the test so that we can catch the bug fast when the test fails, and sometimes we may find out something weird when the log contains unexpected result.